adjust docs for access function by external address (#220)
OpenFunction/openfunction.dev: content/en/docs/concepts/networking/function-entrypoints.md

### Access functions by the external address

> First, you should choose and configure a DNS based on your network environment:

Reviewer: To access a sync function by the external address, you'll need to configure DNS first. Either Magic DNS or real DNS works:

- Magic DNS

  Get Kubernetes Gateway's ip address:
  ```shell
  export IP=$(kubectl get node -l "! node.kubernetes.io/exclude-from-external-load-balancers" -o=jsonpath='{.items[0].status.addresses[?(@.type=="InternalIP")].address}')
  ```
  Replace domain defined in OpenFunction Gateway with Magic DNS:
  ```shell
  export DOMAIN="$IP.sslip.io"
  kubectl patch gateway.networking.openfunction.io/openfunction -n openfunction --type merge --patch '{"spec": {"domain": "'$DOMAIN'"}}'
  ```
  
- Real DNS

>   If your networking environment produced an External IP address, then configure a wildcard A record for the domain:

Reviewer: ```suggestion
  If you have an external IP address, you can configure a wildcard A record as your domain:
```

  ```
  # Here example.com is the domain defined in OpenFunction Gateway
  *.example.com == A <external-ip>
  ```

>   If your networking environment produced a CNAME, then configure a CNAME record for the domain:

Reviewer: ```suggestion
  If you have a CNAME, you can configure a CNAME record as your domain:
```

  ```
  # Here example.com is the domain defined in OpenFunction Gateway

>   *.example.com == A <external-name>

Reviewer: `A <external-name>` ?

  ```

> 

Reviewer: Missing steps to configure openfunction domain for the real DNS method?

Then, you can get `Function` external address by running following command:
```shell
export FUNC_EXTERNAL_ADDRESS=$(kubectl get function function-sample -o=jsonpath='{.status.addresses[?(@.type=="External")].value}')
```

> 

Reviewer: We need examples of function CR status containing `external address` for both magic DNS and real DNS method here to be more intuitive in yaml format.
